app/javascript/app/data/constants.js

```diff
@@ -374,6 +375,9 @@ export const DATA_EXPLORER_TO_MODULES_PARAMS = {
     },
     categories: {
       key: 'category'
+    },
+    subcategories: {
+      key: 'subcategories'
```

Comment:
This should be key: 'subcategory' to be parsed correctly

Reply:
I'll change it to prepare the next iteration but (as for now) the module (pathways page) is not accepting `subcategory` as URL param.

Reply:
⚡️ done